src/items/cult-item/cultSheet.ts

```diff
@@ -91,6 +91,5 @@ export class CultSheet extends RqgItemSheet<ItemSheet.Options, CultSheetData | I
 
   protected async _onDrop(event: DragEvent): Promise<void> {
     super._onDrop(event);
-    await droppableJournalDescription(this.item, event);
   }
```

Comment:
This override is not necessary anymore I think.

Reply:
Did you remove this and the other one below, because I don't have them anymore?  I also thought I got them all with the big commit.